### PR TITLE
added model: Bet and Gain, added controller: ConsoleHandler, added da…

### PR DESCRIPTION
**Pull request by mfedenko09**
…o: BetDao, GainDao and implementation for them: BetDaoImp and GainDaoImpl, added db: Storage and Maib class

### CODE REVIEW

**Review by oleksandrdziubiuk — APPROVED** [review]

File: src/main/java/core/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package core.controller;
+
+import core.dao.BetDao;
+import core.dao.BetDaoImp;
+import core.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImp();
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/controller/ConsoleHandler.java]: access modifier

Comment (dmytro-hryhoruk) [reply src/main/java/core/controller/ConsoleHandler.java]: other Dao?

File: src/main/java/core/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        while (true) {
+            String input = scanner.nextLine();
+            if (input.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = input.split(" ");
+                bet = new Bet(Integer.parseInt(betData[0]), Double.parseDouble(betData[1]));
+            } catch (NumberFormatException e) {
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/controller/ConsoleHandler.java]: Try to enter only one word

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/core/controller/ConsoleHandler.java (hunk 2 above)
Comment (dmytro-hryhoruk) [reply src/main/java/core/controller/ConsoleHandler.java]: So what will happen if you will enter only one world?

File: src/main/java/core/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+                String[] strings = input.split(" ");
+                int inputBet = Integer.parseInt(strings[0]);
+                double inputRisk = Double.parseDouble(strings[1]);
+                bet = new Bet(inputBet, inputRisk);
+                gain = new Gain(inputBet * inputRisk);
+            } catch (NumberFormatException e) {
+                System.out.println("Input correct data, please");
+            }
+            gainDao.add(gain);
+            betDao.add(bet);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/controller/ConsoleHandler.java]: Let's add gain and bet only if there are not null

File: src/main/java/core/controller/ConsoleHandler.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            System.out.println((bet == null ? null : bet.toString()) + ", "
+                    + (gain == null ? null : gain.toString()));
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/controller/ConsoleHandler.java]: these checks and calling toString() is done automatically by System.out.println (check it)

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**